app/src/main/java/com/example/se1_hw/SignInActivity.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.se1_hw
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import com.example.se1_hw.databinding.ActivityMainBinding
+
+class SignInActivity : AppCompatActivity() {
+    lateinit var binding:ActivityMainBinding
+    private val REQUEST_CODE = 100
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        Log.d("onCreate", "onCreate(run)")
+        setContentView(binding.root)
+        initButtonClickEvent()
+        inittextClickEvent()
+    }
+    private fun initButtonClickEvent() {
+        binding.button.setOnClickListener{
+            val userID = binding.idedit.text
+            val userPWD = binding.passedit.text
+            if (userID.isNullOrBlank() || userPWD.isNullOrBlank()) {
+                Toast.makeText(
+                        this@SignInActivity,
+                        "아이디/비밀번호를 확인해주세요!",
+                        Toast.LENGTH_SHORT
+                ).show()
+            } else {
+                val intent = Intent(this@SignInActivity, HomeActivity::class.java)
+                startActivity(intent)
+                Toast.makeText(
+                        this@SignInActivity,
+                        "로그인 성공",
+                        Toast.LENGTH_SHORT
+                ).show()
+            }
+
+        }
+    }
+    private fun inittextClickEvent(){
+        binding.signup.setOnClickListener{
```

Comment:
```suggestion
       binding.buttonSignup.setOnClickListener{
```
예진아아 여기서 signup 아이디가 정확히 버튼인지, 텍스트뷰인지, 나중에 봤을 때, 뭔지 잘 모를 수도 있잖아!! 그러니까 내가 위에 써준 코드처럼 아이디는 길어도, 최대한 알아볼 수 있도록 써주는게 좋아 !! 이것도 코틀린 코드 컨벤션 중 하나인데, 차차 지켜나가면 될 것 같아요 ~~ 👍🏻👍🏻 그리고 레이아웃에서 아이디 값을 정해줄 때는 항상 스네이크 케이스 표기법으로 !!!! 예를 들어, button_home_login 이런식으로 언더바, 소문자로 ㅎㅎ

Reply:
오홍 레이아웃에서 아이디 값을 정해줄 때는 항상 스네이크 케이스 표기법 이용하기 ... 끄적끄적... 다음부터는 주의해서 작성할게요~~!ㅎㅎ